src/reactionsystem.jl

```diff
@@ -945,6 +945,17 @@ function get_metadata(reaction::Reaction, md_key::Symbol)
     return metadata[findfirst(isequal(md_key, entry[1]) for entry in get_metadata_dict(reaction))][2]
 end
 
+"""
+    get_noise_scaling(reaction::Reaction)
+
+Retrives the nosie scaling factor for the reaction's noise term (as generated by the reaction in an SDE). 
+If a specific noise scaling expression have been provided in the reactions "noise_scaling" metadata,
+returns that. If not, returns `1.0`.
+"""
+function get_noise_scaling(reaction::Reaction)
+    has_metadata(reaction, :noise_scaling) ?  get_metadata(reaction, :noise_scaling) : 1.0
+end
```

Comment:
The approach should be to have a separate `has_noise_scaling` function and check that is made, and one then only calls this when a noise scaling exists (otherwise giving an error).

Reply:
That is more consistent with how we and MTK handle other optional fields.